### PR TITLE
Refine sample graphs, adjust styles

### PR DESCRIPTION
**Pull request by songofroland**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -16,17 +16,15 @@
...
+  height: 70vmin;
+  width: 70vmin;
 }
 
 .node {
   height: 2vh;
   width: 2vh;
   border-radius: 50%;
-  font-size: 2vh;
+  font-size: 1.65vh;
```
Comment (wilmhit) [inline src/App.css:27]: Font sizes here should be relative to canvas size, so I would suggest `font-size: 1vmin` in `.canvas` and here `1.65em`

File: src/App.css [hunk 2]
```diff
@@ -36,6 +34,8 @@
 .edge-label {
   height: 100%;
   width: 100%;
+  margin-top: -2vh;
+  font-size: 2.5vh;
```
Comment (wilmhit) [inline src/App.css:38]: same here

File: src/components/CanvasAnimator.tsx [hunk 3]
```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useState } from 'react';
 import { Frame, GraphIndex, LabeledGraph } from '../logic/commonInterfaces';
 import Canvas from './Canvas';
 
-const MS_PER_FRAME = 1000;
+const MS_PER_FRAME = 1650;
```
Comment (wilmhit) [inline src/components/CanvasAnimator.tsx:5]: Thats very precise

File: src/App.css [hunk 4]
```diff
@@ -16,17 +16,15 @@
 .canvas {
   background-color: var(--element-background);
   position: relative;
-  height: 60vmin;
-  width: 60vmin;
-  min-height: 60vmin;
-  min-width: 60vmin;
+  height: 70vmin;
+  width: 70vmin;
```
Comment (wilmhit) [inline src/App.css:20]: 70 does not fit the screen along with graph-editor without scrolling

Comment (wilmhit) [reply src/App.css:20]: Imo it would be better if canvas and graph editor was in a row on desktop.

File: src/logic/graphSamples.ts [hunk 5]
```diff
@@ -12,22 +12,24 @@ const samples: { [key: string]: any } = {
...
-      [1, 1, 0, 1], 
-      [1, 1, 1, 0],
+      [0, 2, 6, 0, 0],
+      [2, 0, 0, 5, 0],
+      [6, 0, 0, 8, 0],
+      [0, 5, 8, 0, 10],
+      [0, 0, 0, 10, 0],
     ],
-    string: '0 1 1 1\n1 0 1 1\n1 1 0 1\n1 1 1 0',
+    string: '0 2 6 0 0\n2 0 0 5 0\n6 0 0 8 0\n0 5 8 0 10\n0 0 0 10 0',
```
Comment (wilmhit) [inline src/logic/graphSamples.ts:23]: There should be a function to do this.

**Review by wilmhit — CHANGES_REQUESTED** [review]
Just do the things you have time for

File: src/components/CanvasAnimator.tsx (hunk 3 above)
Comment (songofroland) [reply src/components/CanvasAnimator.tsx:5]: Yes, I found the best possible empirically 🙂

File: src/App.css (hunk 4 above)
Comment (songofroland) [reply src/App.css:20]: I'll go back to 60 then, life is too short to mess with CSS

File: src/logic/graphSamples.ts (hunk 5 above)
Comment (songofroland) [reply src/logic/graphSamples.ts:23]: Right, let's add that the next time when some of us will be adjusting it

**Review by wilmhit — APPROVED** [review]

**Pull request merged**